.github/workflows/expo-update.yml

```
@@ -0,0 +1,64 @@
+# Tested on MacOS with:
+# act -s GITHUB_TOKEN="$(gh auth token)" release --container-architecture linux/amd64 -P ubuntu-latest=catthehacker/ubuntu:act-latest -e .github/workflows/expo-update-test.json
+name: Update whisper-kit-expo
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  update-whisperkit:
+    runs-on: ubuntu-latest
+    env: 
+      TAG: ${{ github.event.release.tag_name }}
+      BRANCH_NAME: update-whisperkit-${{ github.event.release.tag_name }}
+    steps:
+        - name: Checkout whisper-kit-expo
+          uses: actions/checkout@v4
+          with:
+            repository: seb-sep/whisper-kit-expo
+            fetch-depth: 0
+            token: ${{ secrets.GITHUB_TOKEN }}
+            ref: main
+        
+        - name: Setup Node
+          uses: actions/setup-node@v4
+          with:
+            node-version: '20.x'
+
+        - name: Configure Git
+          run: |
+            git config --global user.name 'github-actions'
+            git config --global user.email 'github-actions@github.com'
+
+        - name: New branch
+          run: |
+            git checkout -b $BRANCH_NAME
+            echo ${{ github.event.release }}
+            echo "Release tag is $TAG"
+        
+        - name: Run update script
+          run: node ./scripts/update-whisperkit.js $TAG
```

This is potentially a security issue without verifying what is in this js file ahead of time with a checksum -  the easiest would be to just have the code running directly here in the workflow. Here is a possible solution (please verify)
```suggestion
        - name: Update package.json version
          run: |
            PACKAGE_PATH="package.json"

            # Check if the package.json exists at the specified path
            if [ ! -f "$PACKAGE_PATH" ]; then
              echo "Could not find package.json at path: $PACKAGE_PATH."
              exit 1
            fi

            # Normalize the release tag by removing a leading 'v', if present
            RELEASE_TAG=${TAG#v}

            # Update package.json with the new version
            jq --arg newver "$RELEASE_TAG" '.whisperKit.version = $newver' "$PACKAGE_PATH" > tmp.$$.json && mv tmp.$$.json "$PACKAGE_PATH"

            # Verify changes
            cat "$PACKAGE_PATH"
```